dhis-2/dhis-api/src/test/java/org/hisp/dhis/category/CategoryOptionComboTest.java

```diff
@@ -145,13 +145,13 @@ void hasDefault() {
 
   @Test
   void testIsDefault() {
-    categoryComboA.setName(CategoryCombo.DEFAULT_CATEGORY_COMBO_NAME);
+    optionComboA.setName(CategoryOptionCombo.DEFAULT_NAME);
     assertTrue(optionComboA.isDefault());
   }
 
   @Test
   void testIsNotDefault() {
-    categoryComboA.setName(CategoryCombo.DEFAULT_CATEGORY_COMBO_NAME + "x");
```

Comment:
This was correct before the change. For a `CategoryCombo` you should use `CategoryCombo.DEFAULT_CATEGORY_COMBO_NAME`.